core/src/test/java/org/infinispan/container/offheap/OffHeapBoundedMemoryTest.java

```diff
@@ -96,6 +98,59 @@ public void testAllocatedAmountEqualWithVersion() {
       assertEquals(allocator.getAllocatedAmount(), container.currentSize);
    }
 
+   public void testAllocatedAmountEqualWithExpiration() {
+      ConfigurationBuilder builder = new ConfigurationBuilder();
+      builder.memory()
+            .size(MemoryUnit.MEGABYTES.toBytes(20))
```

Comment:
Why not 2MB?

Reply:
That is just what the others are using, tbh the value doesn't matter as it is never hit :)

Reply:
Ah right, I was thinking that off-heap preallocates the memory. Since only the `MemoryAddressHash` table is preallocated and that doesn't change with the memory size, I guess it doesn't matter.